src/components/cards/PairPriceGraph/PairPriceGraph.vue

```diff
@@ -0,0 +1,323 @@
+<script setup lang="ts">
+import { computed, reactive, ref } from 'vue';
+import anime from 'animejs';
+import { useStore } from 'vuex';
+import useTokens from '@/composables/useTokens';
+import { coingeckoService } from '@/services/coingecko/coingecko.service';
+import { useQuery } from 'vue-query';
+import {
+  Dictionary,
+  mapKeys,
+  mapValues,
+  maxBy,
+  minBy,
+  pickBy,
+  toPairs
+} from 'lodash';
+import { fromUnixTime, format } from 'date-fns';
+import useTailwind from '@/composables/useTailwind';
+import useBreakpoints from '@/composables/useBreakpoints';
+import { useTradeState } from '@/composables/trade/useTradeState';
+import { getAddress } from '@ethersproject/address';
+const easing = 'spring(1, 150, 18, 0)';
```

Comment:
minor - but would be great to insert a new line after the last `import`

Reply:
ye